src/main/java/sc/fiji/bdvpg/spimdata/importer/SpimDataFromXmlImporterAndRegisterer.java

```diff
@@ -1,44 +1,42 @@
 package sc.fiji.bdvpg.spimdata.importer;
 
 import bdv.spimdata.XmlIoSpimDataMinimal;
-import mpicbg.spim.data.SpimData;
 import mpicbg.spim.data.SpimDataException;
-import mpicbg.spim.data.XmlIoSpimData;
 import mpicbg.spim.data.generic.AbstractSpimData;
 import sc.fiji.bdvpg.services.BdvService;
 
 import java.io.File;
 import java.util.function.Function;
 
-public class SpimDataImporterXML implements Runnable, Function<File, AbstractSpimData> {
+public class SpimDataFromXmlImporterAndRegisterer implements Runnable, Function<File, AbstractSpimData> {
```

Comment:
Registerer is a bit overkill. Also this would mean that we have to add this name in many actions. Maybe we can avoid this ?

Reply:
Ok, I see. I changed it back for now in https://github.com/bigdataviewer/bigdataviewer-playground/pull/66/commits/4ef61ca423cd2e79db842dfa9d1d5f55f05a9e88

Reply:
Maybe we have a smart idea later how to do this better.